src/Examples/Advanced/CustomProperties/Parameter_estimation_PR.ipynb

```diff
@@ -0,0 +1,950 @@
+   "source": [
+    "## 3.0 Setting up an initialized model\n",
+    "\n",
+    "We need to provide a method that returns an initialized model to the `parmest` tool in Pyomo.\n",
+    "\n",
+    "How we build the model will depend on the data we provided in the data dataframe from pir .csv file.\n",
+    "\n",
+    "In this case we have data on the liquid mixture, the temperature and the pressure. We will fix the temperature, mole franction in the liquid phase, and the mole fraction of the inlet. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def PR_model(data):\n",
+    "    \n",
+    "    m = ConcreteModel()\n",
+    "\n",
+    "    m.fs = FlowsheetBlock(default={\"dynamic\": False})\n",
+    "\n",
+    "    m.fs.properties = GenericParameterBlock(default=configuration)\n",
+    "\n",
+    "    m.fs.state_block = m.fs.properties.build_state_block([1],\n",
+    "        default={\"defined_state\": True})\n",
+    "\n",
+    "    m.fs.state_block[1].flow_mol.fix(1)\n",
+    "    m.fs.state_block[1].temperature.fix(298.15)\n",
+    "    m.fs.state_block[1].pressure.fix(1014386)\n",
+    "    m.fs.state_block[1].mole_frac_comp[\"bmimPF6\"].fix(0.16)\n",
+    "    m.fs.state_block[1].mole_frac_comp[\"carbon_dioxide\"].fix(0.84)\n",
+    "    \n",
+    "    # parameter - kappa_ij (set at 0.3, 0 if i=j)\n",
+    "    m.fs.properties.PR_kappa['bmimPF6', 'bmimPF6'].fix(0)\n",
+    "    m.fs.properties.PR_kappa['bmimPF6', 'carbon_dioxide'].fix(-0.4)\n",
+    "    m.fs.properties.PR_kappa['carbon_dioxide', 'carbon_dioxide'].fix(0)\n",
+    "    m.fs.properties.PR_kappa['carbon_dioxide', 'bmimPF6'].fix(0.02)\n",
+    "\n",
+    "    # Initialize the flash unit\n",
+    "    m.fs.state_block.initialize(outlvl=idaeslog.INFO)\n",
+    "\n",
+    "    # Fix the state variables on the state block\n",
+    "    m.fs.state_block[1].pressure.unfix()\n",
+    "    m.fs.state_block[1].temperature.fix(float(data[\"temperature\"]))\n",
+    "    m.fs.state_block[1].mole_frac_phase_comp['Liq', 'bmimPF6'].fix(float(data[\"x_bmimPF6\"]))\n",
+    "    m.fs.state_block[1].mole_frac_phase_comp['Liq', 'carbon_dioxide'].fix(float(data[\"x_carbon_dioxide\"]))\n",
+    "    m.fs.state_block[1].mole_frac_comp['bmimPF6'].fix(float(data[\"x_bmimPF6\"]))\n",
+    "    m.fs.state_block[1].mole_frac_comp['carbon_dioxide'].unfix()\n",
+    "    # Set bounds on variables to be estimated\n",
+    "    m.fs.properties.PR_kappa['bmimPF6', 'carbon_dioxide'].setlb(-5)\n",
+    "    m.fs.properties.PR_kappa['bmimPF6', 'carbon_dioxide'].setub(5)\n",
+    "\n",
+    "    m.fs.properties.PR_kappa['carbon_dioxide', 'bmimPF6'].setlb(-5)\n",
+    "    m.fs.properties.PR_kappa['carbon_dioxide', 'bmimPF6'].setub(5)\n",
+    "\n",
+    "    # Return initialized flash model\n",
+    "    return m"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3.1 Solving square problem"
```

Comment:
This example fails for me at this point, with an infeasible solution. This might be due to changes in how scaling is implemented from recent PRs, although I tried quickly implementing the scaling tools which failed to fix the problem.

Reply:
I will change the initialization method to initialize each point at the experimental composition. Also, I believe I may be adding a problem with the new scaling by scaling the objective function.